worksheet_D/oxo.py

```diff
@@ -0,0 +1,99 @@
+class OxoBoard:
+    def __init__(self):
+        self.gameBoard = {(0, 0):0, (0, 1):0, (0, 2):0,                 # Creates dictionary data structure.

```

Comment:
PEP-8 specifies that variables and fields should use `lower_case_with_underscores` naming convention